functions/dataManger/collectionManager.ts

```diff
@@ -0,0 +1,71 @@
+const AWS = require('aws-sdk');
+const isTest = process.env.JEST_WORKER_ID;
+const config = {
+  convertEmptyValues: true,
+  ...(isTest && {
+    endpoint: 'localhost:8000',
+    sslEnabled: false,
+    region: 'local-env',
+  }),
+};
+const docClient = new AWS.DynamoDB.DocumentClient(config);
+const table = (isTest) ? process.env.DYNAMO_DB_TEST_TABLE : process.env.DYNAMO_DB_GAME_TABLE;
+import { Game } from "../models/game";
+import { createGame, modifyGame, deleteGame, serializeDynamoResponse } from "./gameManager";
+import { Collection } from "../models/collection";
+import * as Interfaces from "../interfaces/interfaces";
+
+export async function getCollection(collection: Collection) : Promise<[Game]> {
+  let params = {
+    TableName: table,
+    KeyConditionExpression: `#partitionKey = :partitionKey AND begins_with(sortKey, :sortKey)`,
+    ExpressionAttributeNames: {
+        "#partitionKey": "partitionKey",
+    },
+    ExpressionAttributeValues: {
+        ":partitionKey": collection.partitionKey,
+        ":sortKey": `[CollectionItem]#[${collection.collectionType}]#[GameItem]`
+    }
+  };
+  
+  try {
+    let response = await docClient.query(params).promise();      
+    let gameList = [] as any
+    response.Items.forEach((game: Interfaces.IDynamoObject) => {
```

Comment:
might want to think about taking advantage of pagination on the query to DDB and implementing it on the API. (We can talk about this more)